drf_spectacular/views.py

```diff
@@ -42,9 +51,15 @@ def get(self, request, *args, **kwargs):
             ModuleWrapper = namedtuple('ModuleWrapper', ['urlpatterns'])
             self.urlconf = ModuleWrapper(tuple(self.urlconf))
 
+        if settings.USE_I18N and request.GET.get('lang'):
+            with translation.override(request.GET.get('lang')):
+                return self._get_schema_response(request)
+        else:
+            return self._get_schema_response(request)
+
+    def _get_schema_response(self, request):
         generator = self.generator_class(urlconf=self.urlconf, api_version=self.api_version)
-        schema = generator.get_schema(request=request, public=self.serve_public)
-        return Response(schema)
+        return Response(generator.get_schema(request=request, public=self.serve_public))
```

Comment:
unrelated/unnecessary change here, if that matters to you, and didnt have a good reason for it.

Reply:
no particular reason. just looked cleaner to me in that spot